src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1563,6 +1577,118 @@ def _get_linux_multicontainer_encoded_config_from_file(file_name):
     # Decode base64 encoded byte array into string
     return b64encode(config_file_bytes).decode('utf-8')
 
+def update_deployment_configs(cmd, resource_group_name, name,
+                              deployment_storage_name=None,
+                              deployment_storage_container_name=None, deployment_storage_auth_type=None,
+                              deployment_storage_auth_value=None, slot=None):
+    
+    # logger.warning("deployment_storage_name: %s deployment_storage_container_name: %s", deployment_storage_name, deployment_storage_container_name)
+    logger.warning("deployment_storage_auth_type: %s deployment_storage_auth_value: %s", deployment_storage_auth_type, deployment_storage_auth_value)
+    
+    if (deployment_storage_name == None or deployment_storage_container_name == None):
+        raise ArgumentUsageError("Please provide both --deployment-storage-name and --deployment-storage-container-name.")
+    
+    if deployment_storage_auth_type == 'userAssignedIdentity' and not deployment_storage_auth_value:
+        raise ArgumentUsageError('--deployment-storage-auth-value is required when --deployment-storage-auth-type is set to userAssignedIdentity.')
+    
+    if deployment_storage_auth_value and deployment_storage_auth_type != 'userAssignedIdentity':
+        raise ArgumentUsageError(
+            '--deployment-storage-auth-value is only a valid input when --deployment-storage-auth-type set to userAssignedIdentity. '
+            'Please try again with --deployment-storage-auth-type set to userAssignedIdentity.'
+        )
+
+    # NameValuePair = cmd.get_models('NameValuePair')
+    NameValuePair = cmd.get_models('NameValuePair')
+    
+    is_flex = is_flex_functionapp(cmd.cli_ctx, resource_group_name, name) 
+
+    if not is_flex:
+        raise ValidationError("This command is only valid for Azure Functions on the FlexConsumption plan.")
```

Comment:
Instead of doing this here, in the commands.py file, you can make changes similar to [this](https://github.com/kamperiadis/azure-cli/blob/10f345d049ecdaf709ea93820041a994fe8a8000/src/azure-cli/azure/cli/command_modules/appservice/commands.py#L347)

Reply:
done.